### PR TITLE
api: correct proposed swaps

### PR DESCRIPTION
**Pull request by dcposch**
- load from daimo_transfers in orders
- correct proposedSwap timestamps
- better logging

### CODE REVIEW

File: packages/daimo-api/src/contract/foreignCoinIndexer.ts [hunk 1]
```diff
@@ -78,6 +79,8 @@ export class ForeignCoinIndexer extends Indexer {
           ])
           .where("chain_id", "=", chainConfig.chainL2.id)
           .where((eb) => eb.between("block_num", "" + from, "" + to))
+          .orderBy("block_num", "asc")
+          .orderBy("sort_idx", "asc")
```
Comment (andrewliu08) [inline packages/daimo-api/src/contract/foreignCoinIndexer.ts:83]: does this use the index on the table? or do we need to do
```
.orderBy("block_num", "asc")
.orderBy("tx_idx", "asc")
.orderBy("sort_idx", "asc")

File: packages/daimo-api/src/contract/homeCoinIndexer.ts [hunk 2]
```diff
@@ -84,6 +84,8 @@ export class HomeCoinIndexer extends Indexer {
           .where("chain_id", "=", chainConfig.chainL2.id)
           .where((e) => e.between("block_num", "" + from, "" + to))
           .where("token", "=", hexToBuffer(chainConfig.tokenAddress))
+          .orderBy("block_num", "asc")
+          .orderBy("sort_idx", "asc")
```
Comment (andrewliu08) [inline packages/daimo-api/src/contract/homeCoinIndexer.ts:88]: same here

**Review by andrewliu08 — APPROVED** [review]
lgtm

File: packages/daimo-api/src/contract/foreignCoinIndexer.ts (hunk 1 above)
Comment (dcposch) [reply packages/daimo-api/src/contract/foreignCoinIndexer.ts:83]: order by is nbd (=sorting a small result set), important part is that the WHERE clauses are a prefix of the index

**Pull request merged**